frontend/src/stop.ts

```diff
@@ -0,0 +1,26 @@
+import type { Directive } from "vue";
+
+/** directive that stops all event propagations on an element */
```

Comment:
Needed to do this for map legends. Without it, click/drag/zoom/etc events done on the legend will propagate down into the tile layer and scroll/zoom/etc.